library/core/src/num/int_macros.rs

```diff
@@ -1383,11 +1386,12 @@ macro_rules! int_impl {
         #[must_use = "this returns the result of the operation, \
                       without modifying the original"]
         #[inline(always)]
+        #[rustc_allow_const_fn_unstable(const_inherent_unchecked_arith)]
```

Comment:
annot: this was previously using the intrinsic on stable, so the `rustc_allow_const_fn_unstable` isn't exposing anything new to stable here.